Makefile

```diff
@@ -365,6 +370,18 @@ e2e-test: e2e-bootstrap e2e-helm-deploy # run test for windows
 e2e-teardown: $(HELM)
 	helm delete csi-secrets-store --namespace kube-system
 
+.PHONY: e2e-deploy-manifest
+e2e-deploy-manifest:
```

Comment:
If you want to test this without the https://github.com/kubernetes/test-infra/pull/23054 changes, you can update this target to `e2e-helm-deploy` and see the results.

Reply:
It [works](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_secrets-store-csi-driver/669/pull-secrets-store-csi-driver-e2e-azure/1422620669878734848).